services/newApiModelSync/scheduler.ts

```
@@ -0,0 +1,366 @@
+
+/**
+ * Message handler for New API Model Sync
+ */
+export const handleNewApiModelSyncMessage = async (
+  request: any,
+  sendResponse: (response: any) => void
+) => {
+  try {
+    switch (request.action) {
+      case "newApiModelSync:triggerAll":
+        const resultAll = await newApiModelSyncScheduler.executeSync()
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)